provision-vm-01.sh

```diff
@@ -59,16 +66,16 @@ function add_message {
 	echo "$MESSAGE" | tee /etc/motd > /dev/null
 
 	echo "$BANNER" | tee /etc/issue > /dev/null
-
-	echo "Banner /etc/issue" | tee -a /etc/ssh/sshd_config > /dev/null
-	systemctl restart sshd
-
+	if ! grep -q "Banner /etc/issue" /etc/ssh/sshd_config; then
+		echo "Banner /etc/issue" | tee -a /etc/ssh/sshd_config > /dev/null
+		systemctl restart sshd
+	fi
```

Comment:
Added ssh banner link (placed in /etc/ssh/sshd_config) check (problem was described in issue #3)